core/src/test/java/com/sequenceiq/cloudbreak/repository/StackUpdaterTest.java

```diff
@@ -0,0 +1,106 @@
+package com.sequenceiq.cloudbreak.repository;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.AdditionalAnswers.returnsFirstArg;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyList;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.sequenceiq.cloudbreak.TestUtil;
+import com.sequenceiq.cloudbreak.domain.Resource;
+import com.sequenceiq.cloudbreak.domain.Stack;
+import com.sequenceiq.cloudbreak.domain.Status;
+import com.sequenceiq.cloudbreak.service.events.CloudbreakEventService;
+
+@RunWith(MockitoJUnitRunner.class)
+public class StackUpdaterTest {
+
+    @Mock
+    private StackRepository stackRepository;
+
+    @Mock
+    private CloudbreakEventService cloudbreakEventService;
+
+    @Mock
+    private ResourceRepository resourceRepository;
+
+    @InjectMocks
+    private StackUpdater underTest;
+
+    @Test
+    public void updateStackStatusWithoutStatusReasonThenNoNotificationSentOnWebsocket() {
+        Stack stack = TestUtil.stack();
+
+        when(stackRepository.findById(anyLong())).thenReturn(stack);
+        when(stackRepository.save(any(Stack.class))).thenReturn(stack);
+        doNothing().when(cloudbreakEventService).fireCloudbreakEvent(anyLong(), anyString(), anyString());
+
+        underTest.updateStackStatus(1L, Status.DELETE_COMPLETED);
```

Comment:
Is the stack instance updated by the method being tested? If so assign the method return value to it; otherwise i dont't see what's being tested :)